### PR TITLE
Clarify exclude/include behavior in README

### LINKED ISSUE #2155: dartdoc_options exclude ignored
On a Flutter package, when I exclude packages via cmd line flag I get the expected behavior, if I move that list to options the exclude is ignored.

```yaml
dartdoc:
  exclude: ["dart:async", "dart:collection", "dart:convert", "dart:core", "dart:developer", "dart:io", "dart:isolate", "dart:math", "dart:typed_data", "dart:ui", "dart:ffi", "dart:html", "dart:js", "dart:js_util"]
```

I tried adding some other option to the file just to be sure it was actually used by dartdoc and I can see it is actually used and other options work fine.

### PR DESCRIPTION
**Pull request by jcollins-g**
Fixes #2155.

Probably not P1 anymore since the underlying issue causing people to try to use the exclude parameter in this way is fixed, but this PR updates the README to hopefully eliminate remaining confusion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,18 +120,23 @@ dartdoc:
     - tool-error
 ```
 
+#### dartdoc_options.yaml fields
+
+In general, **paths are relative** to the directory the `dartdoc_options.yaml` the option is defined in
```
Comment (srawlins) [inline README.md:125]: The grammar here is hard to parse. Maybe

> In general, paths are relative to the directory _of_ the `dartdoc_options.yaml` _file_ _in which_ the option is defined, and should be specified as POSIX paths.

Comment (jcollins-g) [reply README.md]: done
